src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -74,6 +80,10 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
         setOpenNetworkModificationsDialog(false);
     };
 
+    const doDeleteModification = (uuid) => {
+        deleteModification(studyUuid, selectedNode, uuid);
```

Comment:
`selectedNode` will not change when deleting a modification... then how `modifications` state is updated after this ?

Reply:
there is a notification when the modification is deleted, we update on notification

Reply:
Ok, I see.